template_config.yml

```diff
@@ -48,9 +48,8 @@ redmine_project: ansible_plugin
 stable_branch: null
 test_bindings: false
 test_cli: true
-test_fips_nightly: false
 test_performance: false
-test_released_plugin_with_next_pulpcore_release: true
+test_released_plugin_with_next_pulpcore_release: false
```

Comment:
Why is this set to false?

Reply:
it almost never passes
https://github.com/pulp/pulp_ansible/actions.html?page=1&workflow_file_name=nightly.yml

Reply:
Oh ok. I'm not familiar with the job but I guess we can safely ignore it?

Reply:
we've been ignoring all this time, I'm also not very familiar with this job